### PR TITLE
45 client sign up

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to
#45 

=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/services/error-handler/messages.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 const errors = {
   INVALID_USER_ROLE: 'invalid role',
+  INVALID_TOKEN: 'invalid invite link',
```
Comment (thejoefriel) [inline server/services/error-handler/messages.js:3]: Please change this to "This is not a valid invite link to sign up. Please contact your therapist or someone from Chiltern Music Therapy."

File: server/services/error-handler/messages.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
   EMAIL_ALREADY_EXISTS: 'A user account with that email already exists',
   VALIDATION_ERROR: 'error message for more details to user',
   CUSTOM_ERROR_MESSAGE: 'error message for more details to user',
+  NOT_FOUND: 'not found',
```
Comment (thejoefriel) [inline server/services/error-handler/messages.js:7]: Please change to "We cannot find what you're looking for"

File: server/modules/user/model/update.js [hunk 3]
```diff
@@ -1,9 +1,69 @@
...
+      bio = COALESCE($14, u.bio),
+      profile_photo_media_id = COALESCE($15, u.profile_photo_media_id),
+      organisation_id = COALESCE($16, u.organisation_id)
+    WHERE u.id = $1
+    RETURNING *
+  `;
 
-  const res = await query(sql);
+  const res = await query(sql, values, client);
+  console.log(res.rows);
```
Comment (thejoefriel) [inline server/modules/user/model/update.js:65]: please remove the console log

**Review by thejoefriel — CHANGES_REQUESTED** [review]
- [x] Remove text-decoration underline from checkbox input to stop double lines as I think we add border bottom to these links in the styling instead to make the line follow the wireframes 
image

- [ ] Icons are showing pink/red rather than blue: 
image

File: server/services/error-handler/messages.js (hunk 1 above)
Comment (RamyAlshurafa) [reply server/services/error-handler/messages.js:3]: done

File: server/services/error-handler/messages.js (hunk 2 above)
Comment (RamyAlshurafa) [reply server/services/error-handler/messages.js:7]: done

File: server/modules/user/model/update.js (hunk 3 above)
Comment (RamyAlshurafa) [reply server/modules/user/model/update.js:65]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**